Add a configuration section and improve the usage section (#80)
Mogztter/asciidoctor-kroki: README.md

      var kroki = AsciidoctorKroki

      const registry = asciidoctor.Extensions.create()
      kroki.register(registry) // <1>
      var result = asciidoctor.convert(input, {'extension_registry': registry})
      document.getElementById('content').innerHTML = result
    </script>
  </body>
</html>
```
**<1>** Register the extension in a dedicated registry

## Usage

Kroki currently supports the following diagram libraries:

> * ActDiag: `actdiag`

Reviewer: It's probably a bit of effort but as an end user who is not familiar with all of these I would like each one to be a hyperlink to the library location.

* BlockDiag: `blockdiag`
* BPMN: `bpmn`
* Bytefield: `bytefield`
* C4 (PlantUML): `c4plantuml`
* Ditaa: `ditaa`
* ERD: `erd`
* GraphViz : `graphviz`
* Mermaid: `mermaid`
* Nomnoml: `nomnoml`
* NwDiag: `nwdiag`
* PacketDiag: `packetdiag`
* PlantUML: `plantuml`
* RackDiag: `rackdiag`
* SeqDiag: `seqdiag`
* SVGBob: `svgbob`
* UMLet: `umlet`
* Vega: `vega`
* Vega-Lite: `vegalite`
* WaveDrom: `wavedrom`

> In your AsciiDoc document, you can either declare your diagram inline or reference a diagram file using the corresponding macro or using the `include` directive.

Reviewer: ```suggestion
In your AsciiDoc document, you can either write your diagram inline or alternatively you can make a reference to the diagram file using macro form or with the `include` directive.
```
Too many "ors" running together, just a suggestion...

Author: Thanks that's better!

* [ActDiag](https://github.com/blockdiag/actdiag): `actdiag`
* [BlockDiag](https://github.com/blockdiag/blockdiag): `blockdiag`
* [BPMN](https://github.com/bpmn-io/bpmn-js): `bpmn`
* [Bytefield](https://github.com/Deep-Symmetry/bytefield-svg/): `bytefield`
* [C4 (PlantUML)](https://github.com/RicardoNiepel/C4-PlantUML): `c4plantuml`
* [Ditaa](http://ditaa.sourceforge.net): `ditaa`
* [ERD](https://github.com/BurntSushi/erd): `erd`
* [GraphViz](https://www.graphviz.org/): `graphviz`
* [Mermaid](https://github.com/knsv/mermaid): `mermaid`
* [Nomnoml](https://github.com/skanaar/nomnoml): `nomnoml`
* [NwDiag](https://github.com/blockdiag/nwdiag): `nwdiag`
* [PacketDiag](https://github.com/blockdiag/nwdiag): `packetdiag`
* [PlantUML](https://github.com/plantuml/plantuml): `plantuml`
* [RackDiag](https://github.com/blockdiag/nwdiag): `rackdiag`
* [SeqDiag](https://github.com/blockdiag/seqdiag): `seqdiag`
* [SVGBob](https://github.com/ivanceras/svgbob): `svgbob`
* [UMLet](https://github.com/umlet/umlet): `umlet`
* [Vega](https://github.com/vega/vega): `vega`
* [Vega-Lite](https://github.com/vega/vega-lite): `vegalite`
* [WaveDrom](https://github.com/wavedrom/wavedrom): `wavedrom`

> Each diagram libraries support one or more output format.

Reviewer: ```suggestion
Each diagram libraries support one or more output formats.
```

> Consult the [Kroki documentation](https://kroki.io/#support) to find out which format are supported.

Reviewer: ```suggestion
Consult the [Kroki documentation](https://kroki.io/#support) to find out which formats are supported.
```

Author: Good catch! :sweat:

In the example below, we are using the `vegalite` macro to reference a file named *chart.vlite*:

```
vegalite::chart.vlite[svg,role=chart,opts=interactive]
```

![Vega-Lite chart diagram](https://kroki.io/vegalite/png/eNrtVktz2yAQvvtXMEyOqt9pnNz6To-d6c3jA5ZWEg0CF7Ba26P_3gVb2JJSN8mhTWdyMIb92CffCnY9QuiFiXMoGL0hNLd2ZW4GgxIy1s-4zdfLPleD_QYvfSW4hUE57X8zStLI6SdgYs1XlqMAbdwqzbdKWibEhsRKxsyCxF9C4pxpa4jNmSUmVz9IwtMUNEhL7GYFhqgURWgMLN9ymRETMwGmf3DDrItxh3NclUysweB67teE7KjP4A2NCF3ibDyroib0toYuL9vQuxqaTtrQ-xq6HrWhDzU060Afg6-OwU81NLpuQ7fB4FUb-hwMjiuPLHD0m2i-L3Koxe6gSQum75xuzHUsgNYWKchYJVjfUE0v3TSWKEg5iMTpL4Oql7uzcmKpCi6ZaIJGaReJXAvRkLOf3LQcOFM8vnPilAkDURNLVMG4_A1ouRVw8HOCVGFeHRWo4Vt4bHLf10yiE2Z5Ca0MHSnvSaWhiA7_GFashNJ_P65WJbegFeJWr-E04oZpARnI5L7j258C_XI-6d7p_8H0C0v_PUtFhw2aycxtmM-GERm9xmE8xWEyxmE6HC6eJam7afgLy-8oWIZX26OZnSpd-E8qTWh0lvTihfT_C-ltrgHfHaJzpCGf-QR5fjVcnOuK8XDfEM-tF56c3bFZSq45PsDo0y-CryGIhzQFjj4YikpKlMfkOrmGWlIuE1hhEPhqPLbNgUYNMLioelXvF-H7eDo=)

Finally, we can use the `include` directive to reference a diagram file:

```

> [plantuml,alice-bob,svg,role=sequence]

Reviewer: We need to define which attributes are available and what they do somewhere. Perhaps I can do this as part of #72 before too long.

Author: Yes that would be nice :smile:
